### PR TITLE
refactor biz controlling member relationship with Application

### PR DESCRIPTION
**Pull request by peggy-quartech**
# Description

This PR includes the following proposed change(s):

- spdbt-2702(Refactor Controlling member get/update for biz(not for app))

### CODE REVIEW

File: src/Spd.Resource.Repository/BizContact/BizContactRepository.cs [hunk 1]
```diff
@@ -25,13 +25,30 @@ internal class BizContactRepository : IBizContactRepository
 
         public async Task<IEnumerable<BizContactResp>> GetBizAppContactsAsync(BizContactQry qry, CancellationToken ct)
         {
-            IQueryable<spd_businesscontact> bizContacts = _context.spd_businesscontacts;
+            IQueryable<spd_businesscontact> bizContacts = _context.spd_businesscontacts
+                .Expand(c => c.spd_businesscontact_spd_application);
+            if (qry.AppId != null) //change to n:n relationship, so have to do the seperate way.
```
Comment (ruaraya) [inline src/Spd.Resource.Repository/BizContact/BizContactRepository.cs:30]: Please fix typo ("separate")

File: src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs [hunk 2]
```diff
@@ -58,29 +63,38 @@ public async Task ManageBizContactsAsync_WithNoExistingContacts_Correctly()
...
     public async Task ManageBizContactsAsync_WithExistingContacts_Correctly()
     {
+        /********************************************* Dynamics change the schema, licence can only be created when it links to a case. But it seems we cannot create case(incident) in program. So, make this a todo.
+        // So, we need to deal with it later.
         // Arrange
         account biz = await CreateAccountAsync();
         spd_application app = await CreateApplicationAsync(biz);
         contact contact = await CreateContactAsync();
-        spd_licence lic = await CreateLicenceAsync(contact);
+        incident spd_case = await CreateIncidentAsync(contact); //here, we create incident failed. todo: fix it later.
```
Comment (ruaraya) [inline src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs:95]: What is the relationship between "incident" and "case"? the type (`incident`) in this case differs from the variable name (`spd_case`)

File: src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs [hunk 3]
```diff
@@ -58,29 +63,38 @@ public async Task ManageBizContactsAsync_WithNoExistingContacts_Correctly()
...
+            _context.DeleteObject(app);
+            _context.DeleteObject(biz);
+            await _context.SaveChangesAsync(CancellationToken.None);
+        }
     }
 
     [Fact]
     public async Task ManageBizContactsAsync_WithExistingContacts_Correctly()
     {
+        /********************************************* Dynamics change the schema, licence can only be created when it links to a case. But it seems we cannot create case(incident) in program. So, make this a todo.
```
Comment (ruaraya) [inline src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs:89]: Do we have a related ticket for this "to do"?

File: src/Spd.Resource.Repository/BizContact/BizContactRepository.cs (hunk 1 above)
Comment (peggy-quartech) [reply src/Spd.Resource.Repository/BizContact/BizContactRepository.cs:30]: sure

File: src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs (hunk 2 above)
Comment (peggy-quartech) [reply src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs:95]: incident is case in spd.
Dynamics team call it case. But in database, it is incident table.

File: src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs (hunk 3 above)
Comment (peggy-quartech) [reply src/Spd.Resource.Repository.IntegrationTest/BizContactRepositoryTest.cs:89]: https://justice.gov.bc.ca/jira/browse/SPDBT-2716 is created for this.

**Pull request merged**